packages/site/src/layout/NavMenu.module.css

```diff
@@ -0,0 +1,98 @@
+.navMenuContainer {
+    --hoverOffset: var(--space-smallest);
+    --negativeHoverOffset: calc(-1 * var(--hoverOffset));
+    --subItemInset: calc(var(--space-small) + var(--space-smaller));
+    --navItemHeight: 40px;
+    display: flex;
+    flex-direction: column;
+    padding: 36px 0 0 0;
+    height: 100dvh;
+    background-color: var(--color-surface--background);
+    width: 200px;
+    box-sizing: border-box;
+}
+
+.navMenuContainer > * {
+  padding-left: var(--space-base);
+  padding-right: var(--space-base);
+  padding-bottom: var(--space-base);
+}
+
+.navMenuHeader {
+    background-color: var(--color-surface--background);
+    display: flex;
+    flex-direction: column;
+    gap: var(--space-large);
+}
+
+.navMenu {
+    background-color: var(--color-surface--background);
+    color: var(--color-text);
+    display: flex;
+    flex-direction: column;
+    gap: var(--space-large);
+    overflow-y: scroll;
+}
+
+.navMenu li {
+  margin-bottom: var(--space-smaller);
+}
+
+.navMenuItem {
+  align-items: center;
+  display: flex;
+  min-height: var(--navItemHeight);
+  overflow-wrap: anywhere;
+  transition: all var(--timing-base) ease-out;
```

Comment:
other note for future: this is a really standard declaration we set on interactive elements. could we reduce overhead / mental effort by creating a "compound" token or util class that gives you standard interaction stuff Out Of The Box?